protocol/app/upgrades/v0.2.2/constants.go

```diff
@@ -0,0 +1,23 @@
+package v0_2_2
+
+import (
+	store "github.com/cosmos/cosmos-sdk/store/types"
+	"github.com/dydxprotocol/v4-chain/protocol/app/upgrades"
+)
+
+const UpgradeName = "v0.2.2"
```

Comment:
Nit / Non-blocking, I noticed that most teams use whole numbers for versions. Do we plan on using whole numbers for mainnet software or continue to use SemVer in the future?

@roy-dydx